account_invoice_change_currency/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Account Invoice - Change Currency",
+    "version": "14.0.1.0.0",
+    "category": "Accounting & Finance",
+    "summary": "Allows to change currency of Invoice by wizard",
+    "author": "Komit Consulting, Odoo Community Association (OCA)",
```

Comment:
Please add Vauxoo as author (actually that's the original one) and myself as maintainer:

```suggestion
    "author": "Vauxoo, Komit Consulting, Odoo Community Association (OCA)",
    "maintainers": ["luisg123v"],
```

Reply:
Done!